core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/SSHAction.java

```diff
@@ -63,7 +63,11 @@ public SSHAction(SSHActionConfig config) {
   public void run(final ActionContext context) throws Exception {
     Connection connection = new Connection(config.host);
     try {
-      connection.connect();
+      try {
+        connection.connect();
+      } catch (IOException e) {
+        throw new IOException(String.format("Attempt to connect to host %s failed", config.host));
```

Comment:
why do we need to catch and then throw an exception without the original cause? Is there something wrong with the original exception?